client/post-editor/post-editor.jsx

```diff
@@ -848,8 +841,9 @@ const PostEditor = React.createClass( {
 		this.props.receivePost( post );
 
 		// make sure the history entry has the post ID in it, but don't dispatch
+		const site = this.props.sites.getSite( post.site_ID );
 		page.replace(
-			basePath + '/' + this.props.sites.getSite( post.site_ID ).slug + '/' + post.ID,
+			utils.getEditURL( post, site ),
```

Comment:
Can we use the [`getEditorPath` selector](https://github.com/Automattic/wp-calypso/blob/b7297c732f7fbc8651a16219e1a1db10f2699de9/client/state/ui/editor/selectors.js#L68-L86)? I'd rather not duplicate the logic if we can avoid it.